index.html

```diff
@@ -0,0 +1,203 @@
+            <div class="karet">></div>
+            <li>
+              <select id="year-filter">
+                <option value="">Year</option>
+                <option class="year-filter" value="placeholder"></option>
+              </select>
+            </li>
+          </ul>
+        </div>
+        </section>
+        <section id="resource" class="tab-content">
+          <div id="bodyCopy">
+            <h1>Helpful Resources</h1>
+            <p>Owning a car is an important responsibility. Sometimes cars never breakdown, and sometimes they have perpetual issues. We understand that you have a busy life and don't have time to do extensive research on your car's maintenance schedule. That is why we went to the liberty of providing you with a handful of useful resources that you can learn from. <strong>Invest time and research into your car now, so that you don't have to invest money into it later.</strong></p>
+          <div id="articles">
+          <h2>Articles</h2>
+            <ol>
+              <li><a href="http://lifelanes.progressive.com/five-things-to-know-about-your-car/">5 Things Everyone Should Know About Their Car</a></li>
+              <li><a href="http://eliteautorepairtempe.com/10-things-all-car-owners-should-know/">10 Things All Car Owners Should Know</a></li>
+              <li><a href="http://www.advantageautorepairofparker.com/10-things">10 Things You Need to Know About Car Maintenance</a></li>
+            </ol>
+          </div>
+          <div id="videos">
+          <h2>Videos</h2>
+          <p>Are you a visual learner? If so, take a moment to watch the below videos. In no time you will be maintaining your own car and not relying on sneaky mechanics.</p>
+            <div class="videoWrapper">
+              <iframe src="https://www.youtube.com/embed/O1hF25Cowv8" frameborder="0" allowfullscreen></iframe>
+            </div>
+            <div class="videoWrapper">
+              <iframe src="https://www.youtube.com/embed/GaKKSFEsAdI" frameborder="0" allowfullscreen></iframe>
+            </div>
+              <div class="videoWrapper">
+              <iframe src="https://www.youtube.com/embed/s5HGWg-dhPM" frameborder="0" allowfullscreen></iframe>
+            </div>
+          </div>
+        </div>
+        </section>
+        <div id="button-container">
+          <button id="push-to-garage"><a href="/addCar">Store In Garage</a></button>
+        </div>
+
+      <section id="results"></section>
+        <div id="wrapper">
+          <ol id="maintenance"></ol>
+        </div>
+        <div id="button-container">
+          <button id="push-to-garage"><a href="/addCar">Store In Garage</a></button>
+        </div>
+      </section>
+      <div id="root" style="width: 320px; margin: 40px auto; padding: 10px"></div>
+      <script src="https://cdn.auth0.com/js/lock/10.0/lock.min.js"></script>
+      <script>
+         var lock = new Auth0Lock('SRdhH0HiDy8RsMiGkuHTAbFmAWPYO1ZV', 'app55939845.auth0.com', {
```

Comment:
Is this private information? I wonder if making an ajax call to your server.js file, and running through an env var would be a more private way to protect this info and instantiate a `new Auth0Lock` object?